content/en/blog/_posts/2021-07-21-memory-manager-beta.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: blog
+title: "Kubernetes Memory manager moves to beta"
+date: 2021-07-21
+slug: kubernetes-1-22-feature-topoloy-manager-beta
+---
+
+# Memory manager moves to beta
+
+The blog post describes the internals of the memory manager, a beta feature of Kubernetes 1.22. The **Memory Manager** is a component in the **kubelet** ecosystem proposed to enable the feature of guaranteed memory (and hugepages) allocation for pods in the Guaranteed QoS class.
```

Comment:
Optionally, link to https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes

Reply:
done